CHANGE-NOTES.html

```diff
@@ -1,3 +1 @@
-<p>Add support for IntelliJ 2022.2.</p>
-
-<p>Show a path to repository (from project root) instead of a repository name alone in the repository selection combobox.</p>
+<p>Show only VCS error notification when machete file parsing fails (no IDE internal error).</p>
```

Comment:
Actually... should even such a notification be shown? 🤔 isn't a red squiggle in editor enough?

Reply:
Oh! okay, this notification is shown during an attempt to **refresh the repository**, makes sense

Reply:
(not just when machete is being edited and fails to parse... that would be absurd 😅 )